Под ТЗ-6 выполнено следующее:

Vladyak1 opened the pull request:
- HistoryManager дополнен методом remove();
- InMemoryHistoryManager дополнен классом Node, history стал HashMap, новые методы: linkLast(), removeNode(), getAll, остальные методы изменены под HashMap
- InMemoryHistoryManagerTest также изменен под HashMap
- также были внесены изменения в остальные файлы под соответствие файлу checkstyle.xml

Vladyak1 committed:
ТЗ-6 выполнено следующее:
- HistoryManager дополнен методом remove();
- InMemoryHistoryManager дополнен классом Node, history стал HashMap, новые методы: linkLast(), removeNode(), getAll, остальные методы изменены под HashMap
- InMemoryHistoryManagerTest также изменен под HashMap
- также были внесены изменения в остальные файлы под соответствие файлу checkstyle.xml

VladimirTitov4 commented on src/ru/yandex/practicum/tasktracker/service/InMemoryHistoryManager.java:
```diff
@@ -2,37 +2,91 @@
+
+    @Override
+    public Map<Integer, Node> getHistory() {
+        return history;
```
Метод должен возвращать коллекцию задач List<Task>

VladimirTitov4 commented on src/ru/yandex/practicum/tasktracker/service/InMemoryHistoryManager.java:
```diff
@@ -2,37 +2,91 @@
+        }
+        if (nextNode == null) {
+            last = prevNode;
+            //prevNode.next = null;
```
Если код не используется - нужно удалить

Vladyak1 committed:
ТЗ-6 выполнено следующее:
- HistoryManager дополнен методом remove();
- InMemoryHistoryManager дополнен классом Node, history стал HashMap, новые методы: linkLast(), removeNode(), getAll, остальные методы изменены под HashMap
- InMemoryHistoryManagerTest также изменен под HashMap
- также были внесены изменения в остальные файлы под соответствие файлу checkstyle.xml
- удалены лишние закоментированные строки;
- изменён метод getHistory().

Vladyak1 committed:
Delete .idea/checkstyle-idea.xml

Vladyak1 committed:
Delete .idea/checkstyleidea-libs/readme.txt

Vladyak1 committed:
Delete checkstyle.xml

Vladyak1 committed:
Delete java-kanban.iml
